interface/rotg.c

```diff
@@ -112,7 +112,7 @@ void CNAME(FLOAT *DA, FLOAT *DB, FLOAT *C, FLOAT *S){
   if (adb == ZERO) {
     *C = ONE;
     *S = ZERO;
-    DA = ZERO;
+    *DA = ZERO;
```

Comment:
I'm not sure about this change. If the input vector is [a,0], then the rotation is the identity and the rotated vector [a,0]. Maybe your initial intention was to delete the whole line as in the other `#if` branch?

Reply:
yeah, looks like a cut-n-paste gone wrong. thanks for your vigilance - still haven't run any tests with this